environment.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   - matplotlib=3.0.3        # Current 3.1.0 build incompatible with multiqc=1.7
 
   ## bioconda packages
+  - sortmerna=2.1b # for metatranscriptomics
```

Comment:
Could you add a new PR with just this change (+ mini Update of the changelog) to have the docker image updated first before we test this here in this PR using TravisCI? Please follow our guidelines for that directly:

https://nf-co.re/developers/adding_pipelines#adding-new-dependencies-to-an-existing-pipeline

Reply:
Opened pull request #286